wikimedia/wikipedia-ios
# Add compact horizontal sparkline grid to Top read widget

**Phabricator:** https://phabricator.wikimedia.org/T262355

### Notes
Contains some design updates to the Top read widget.

* Adds grid to compact sparkline (visible in small and large sizes)
* Insets compact graph within grid
* Reduces width of small widget sparkline container

### Test Steps
1. Add Top read small and large widgets to iPhone X
2. Ensure horizontal grid lines are visible in both sizes, and that the small widget sparkline container proportionally matches small widget screenshot below

![compact_grid](https://user-images.githubusercontent.com/5652327/93826974-387a1f00-fc1d-11ea-96f2-d5794e6dc127.png)
